app.js

```diff
@@ -0,0 +1,59 @@
+let books = [];
+const addBtn = document.getElementById('add-books');
+
+function displayBooks() {
+  const bookListContainer = document.querySelector('.books-col');
+  bookListContainer.innerHTML = '';
+  books.forEach((book) => {
+    const bookItem = document.createElement('div');
+    bookItem.setAttribute('class', 'book-item');
+    bookItem.innerHTML = `
+      <p class="book-title">${book.title}</p>
+      <p class="book-author">${book.author}</p>
+      <button class="remove-button">Remove</button>
+    `;
+    bookListContainer.appendChild(bookItem);
+  });
+  /* eslint-disable */
+  addRemoveListeners();
+  /* eslint-enable */
+}
+
+function removeBook(title) {
+  books = books.filter((book) => book.title !== title);
+  localStorage.setItem('books', JSON.stringify(books));
+  displayBooks();
+  /* eslint-disable */
+  addRemoveListeners();
+  /* eslint-enable */
+}
+
```

Comment:
- [ ] There is a bug in your app. When I add books of the same title and delete one, all books with that title are deleted. Kindly refactor your remove functionality to remove only the book that was clicked. You can consider using some form of unique ID instead of title as your delete parameter.

  Only the deleted book should be deleted.